demo/pubsub/server/dynamic/subscribe_server.go

```
@@ -0,0 +1,96 @@
+	flag.Parse()
+	if storeName == "" {
+		panic("storeName is empty.")
+	}
+	testDynamicSub()
+}
+
+func testDynamicSub() {
+	cli, err := client.NewClient()
+
+	// Use SubscribeWithHandler API to subscribe to a topic.
+	stop, err := cli.SubscribeWithHandler(context.Background(), client.SubscriptionRequest{
```

golangci-lint

ineffectual assignment to err

(ineffassign)